config/environment.rb

```diff
@@ -1,7 +1,11 @@
 require 'bundler'
 Bundler.require
+require 'pry'
 
-module Concerns
-end
+require_relative "../lib/concerns/findable.rb"
+require_relative "../lib/artist.rb"
+require_relative "../lib/song.rb"
+require_relative "../lib/genre.rb"
+require_relative  "../lib/music_controller.rb"
 
 require_all 'lib'
```

Comment:
You should need all of the extra require statements if `require_all 'lib'` is doing it's job right.